frontend/wallet/src/DaemonProcess.re

```diff
@@ -0,0 +1,45 @@
+open Bindings;
+
+let process: ref(option(ChildProcess.Process.t)) = ref(None);
+
+let start = port => {
+  print_endline("Starting graphql-faker");
+  let p =
```

Comment:
This code finish executing once the process spawns and we don't wait for it to die right? 

Maybe you can have `start` return the callback `kill`. This way, we don't have to use the `ref` process?

Reply:
yeah this asynchronously spawns a process and continues. basically no js operations are blocking